static/robots.txt

```diff
@@ -1,5 +1,5 @@
 User-agent: *
-Allow: /
+Disallow: /
```

Comment:
this file will become not useful since we are deploying to `/docs`

Reply:
OK, you're right. Don't even bother enabling it back after adding the canonical URLs on V1 doc pages.